styles/bsky/catppuccin.user.css

```diff
@@ -599,6 +599,17 @@
     circle[fill="hsl(211, 28%, 24.8%)"][r="1.5"] {
       fill: @surface1 !important;
     }
+
+    /* "save changes" button in edit profile */
+    div[style*="background-image: linear-gradient(94.0544deg, rgb(90, 113, 250), rgb(0, 133, 255))"] {
+      background: @accent-color !important;
+    }
+    div[style*="background-image: linear-gradient(94.0544deg, rgb(90, 113, 250), rgb(0, 133, 255))"] div {
+      color: @crust !important;
+    }
```

Comment:
these two rules can be combined into a single rule by nesting the `div { ... }` part from the second block into the first block
also, this should be moved into the block matching `[data-testid="editProfileSaveBtn"]` that i mentioned in the other comment, where it can be reduced further (as shown in the patch i commented on the PR)